test/models/script_test.rb

```diff
@@ -168,10 +180,13 @@ class ScriptTest < ActiveSupport::TestCase
     @script.requires.clear
     @script.update! name: 'Updated'
 
-    assert_no_difference 'Script.count' do
-      Script.import path
-    end
+    old_count = Script.count
 
+    ret = Script.import path
+
+    assert_equal old_count, Script.count
+    assert_equal 1, ret.count
+    assert ret.first.valid?
```

Comment:
```suggestion
    assert scripts.all?(&:valid?)
```
Así lo dejamos preparado por si en un futuro cambiamos para que importe más de uno, tendríamos que tocar la línea de arriba, pero no esta.

Reply:
excelente